Dockerfile

```diff
@@ -57,6 +57,6 @@ COPY benchmark_runner/main/main.py /benchmark_runner/main/main.py
 
 CMD [ "python3.12", "/benchmark_runner/main/main.py"]
 
-# oc: https://www.ibm.com/docs/en/fci/6.5.1?topic=steps-setting-up-installation-server
-# sudo podman build -t quay.io/benchmark-runner/benchmark-runner:latest . --no-cache
+# How to build the Dockerfile ?
+# sudo podman build --build-arg OCP_CLIENT_VERSION=4.19.0-rc.2 --build-arg VIRTCTL_VERSION=1.5.0 -t quay.io/benchmark-runner/benchmark-runner:latest . --no-cache
```

Comment:
I thought the idea was to pass `VIRTCTL_VERSION` from the secret?

Reply:
I just put it as future example